app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
The `skills` attribute is defined as a class variable. This means it is shared across all instances of `SoftwareEngineer`, which can lead to unexpected behavior when modifying the `skills` list in instances. Consider defining `skills` as an instance variable within the `__init__` method.